src/StateSmithTest/DiagramEmbeddedRenderConfigTests.cs

```diff
@@ -128,7 +128,7 @@ public void CopyDataFromDiagramRenderConfig()
             typedef enum __attribute__((packed)) {enumName}
             """);
 
-            const int expectedOptionCount = 7;
+            const int expectedOptionCount = 8;
```

Comment:
We should update the tests here. I can do if you like.

Update $RenderConfig node here that sets UseStdBool to false.
![image](https://github.com/user-attachments/assets/87aa13cf-1b22-4d4d-ad6b-7463f5f95348)

Add some code like:
```
renderConfig.UseStdBool.Should().BeFalse();
```

Reply:
I made the change.

Reply:
thanks!